com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/flowable/FlowableFacade.java

```diff
@@ -197,6 +197,20 @@ public Execution getProcessExecution(String processInstanceId) {
             .orElse(null);
     }
 
+    public List<HistoricVariableInstance> findHistoricVariablesForProcess(String processInstanceId) {
+        return processEngine.getHistoryService()
+            .createHistoricVariableInstanceQuery()
+            .processInstanceId(processInstanceId)
+            .list();
+    }
+
+    private HistoricProcessInstance getHistoricProcessInstanceById(String processInstanceId) {
```

Comment:
This method should not be necessary after applying my suggestion about `findHistoricProcessInstanceIdByProcessDefinitionIdStartsWith`.